.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
+v                               ✰  Thanks for creating a PR! ✰    
+v    Before smashing the submit button please review the checkboxes.
+v    If a checkbox is n/a - please still include it but + a little note why
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+
+Closes: #XXX
+
+## Description
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review.
+-->
+
+
+______
+
+For contributor use:
+
+- [ ] Wrote tests
+- [ ] Updated CHANGELOG_PENDING.md
```

Comment:
not sure if you will have a pending file or just a changelog with a unreleased section

Reply:
not sure, let's just use the latter.

Reply:
I typically just have an unreleased section at the top of the changelog, but I don't mind either way. What do you think @ancazamfir?

Reply:
unreleased section at top is perfect!